src/System/Params.cpp

```diff
@@ -0,0 +1,74 @@
+#include <System/Params.hpp>
+#include <JSystem/JSupport/JSUMemoryInputStream.hpp>
+
+bool TParams::load(const char* filename)
+{
+	bool found = false;
+	if (filename[0] == '/') {
+		filename++;
+	}
+
+	void* resource = nullptr;
+	if (mSceneArc != nullptr && mSceneArc->becomeCurrent("/map/params")) {
```

Comment:
add a `static const char SceneParamsDir[] = "/map/params";` to the cpp file (don't add a declaration to the hpp, the symbol is local!) and use it here